editor/src/clj/editor/editor_extensions.clj

```diff
@@ -766,7 +767,7 @@
                                .toPath
                                .normalize)
               env (luart/make-env
-                    :find-resource #(find-resource project ec %)
+                    :find-resource #(find-resource project %)
```

Comment:
That `ec` would be a snapshot of the project state at the time the editor scripts were reloaded, so this was quite broken. This fix is also a potential memory savings, as we're no longer holding on to a stale snapshot of the project.